www/static/extension.css

```diff
@@ -1,3 +1,8 @@
+body {
+  background-color: #0d1117;
+  color: #ffffff;
```

Comment:
I think previously the dark mode was only used when the browser preferred it via a `@media (prefers-color-scheme: dark) {` media query. This appears to change it to always use dark mode?